lighthouse-core/audits/audit.js

```diff
@@ -136,7 +137,18 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
+    let auditDescription = audit.meta.description;
 
+    if((result.error || result.rawValue === false) && audit.meta.fallbackDescription) {
```

Comment:
let's use the same logic we do for display, the logic applies if `!result.rawValue || (typeof result.rawValue === 'number' && result.rawValue < 100)`